src/components/StoriesList/index.jsx

```diff
@@ -14,7 +14,7 @@ export default function StoriesList({ stories, images }) {
         return (
           <li key={index}>
             <GatsbyImage
-              image={image?.childrenImageSharp[0].gatsbyImageData}
+              image={image?.childrenImageSharp[0]?.gatsbyImageData}
```

Comment:
Since you've figured out the actual problem, I don't think we need this any more.